.travis.yml

```diff
@@ -6,28 +6,27 @@ php:
   - 7.2
 
 env:
-  - SYMFONY_VERSION=2.7.*
-  - SYMFONY_VERSION=2.8.*
-  - SYMFONY_VERSION=3.0.*
-  - SYMFONY_VERSION=3.1.*
-  - SYMFONY_VERSION=3.2.*
-  - SYMFONY_VERSION=3.3.*
-  - SYMFONY_VERSION=3.4.*
-  - SYMFONY_VERSION=4.0.*
-  - SYMFONY_VERSION=dev-master
+  - SYMFONY_VERSION=lts:^2
+  - SYMFONY_VERSION=lts:^3
+
 
 matrix:
-  exclude:
-    - php: 5.6
-      env: SYMFONY_VERSION=4.0.*
+  fast_finish: true
+  include:
+    - env:
+        - SYMFONY_VERSION=lts:^2
+        - dependencies=lowest
 
+    - env:
+        - SYMFONY_VERSION=lts:^3
+        - cs_fixer=cs_dry_run
+
+  exclude:
     - php: 5.6
-      env: SYMFONY_VERSION=dev-master
+      env: SYMFONY_VERSION=lts:^4
```

Comment:
Symfony 4 is currently not part of the build-matrix. I'm not sure if it your work with a Symfony 4 application, so I guess it is cleanest to just remove version 4 from this file.